harness/pyproject.toml

```diff
@@ -1,4 +1,4 @@
 [tool.black]
 line-length = 100
-exclude = '(_gen.py|determined/_swagger/client/*)'
+extend-exclude = '(_gen.py|determined/_swagger/client/*)'
```

Comment:
I guess this is no worse than before, but `--extend-exclude` completely overrides this, so currently this line would never actually be used. I figure we can either change this to `exclude` and add `build` explicitly or just remove this entirely. Or forget my other suggestion and put it all in one place.

Reply:
Good catch!